data/list.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
its advisable to stick to db.json convention for file name, as this file will not only contain list. Also to obey good practice of REST api, I would name top level key "cities" as opposed to "list". Also best to name this folder with word "mock" and "api" inside, to convey, that this is api related and not a real api for this app, but rather a mocked version of it. I suggest "mocked-api-data" or similar.

Reply:
done 
I renamed folder to "api-data" (not mocked-api) because soon I'm going to delete this default list and it will be "real" data that user added in the form.

Reply:
I'm ok with that, as long as it is a conscious decision. Its worth considering in most real scenarios,  data is stored in a database and not in file system, and api is in a separate git repository. However its up to you to define the architecture of your app.